modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -3,9 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You need to have access to an API in your organization's Exchange to perform this procedure. For example, in Design Center create an API and publish it to Exchange. After setting up the API to manage it from Exchange, the API version, status, and other information appears in API Manager. With this set up, you can configure and manage the API.
+To be able to manage your API that you created using Design Center or any other application, you must first import it to Anypoint Exchange (Exchange). After you create the API in API Manager by importing it from Exchange, you can modify the API version, status, and other information. After saving the configuration, you can deploy the API.
 
-This procedure covers the configuration of the API. After saving the configuration, you can deploy the API.
+*Task Prerequisite*
+
+Before you you perform this procedure, ensure that you have the access permissions to an API in your organization's Exchange.
```

Comment:
```suggestion
Before you import the API from Exchange to API Manager and modify that API instance to suit your needs, ensure that you have the proper API access permissions in your organization's Exchange instance.
```

Reply:
As indicated in earlier edits, I don't think we want to use inline headings like the one you use for prerequisites. 
But Mysti will clarify with her template at some point.

Reply:
Done! Thanks!